diff --git a/apps/dashboard/src/providers/ConfigProvider.tsx b/apps/dashboard/src/providers/ConfigProvider.tsx
index 4de4a82..790e670 100644
--- a/apps/dashboard/src/providers/ConfigProvider.tsx
+++ b/apps/dashboard/src/providers/ConfigProvider.tsx
@@ -7,7 +7,7 @@ import { RoutePath } from '@/enums/RoutePath'
 import { queryKeys } from '@/hooks/queries/queryKeys'
 import { DaytonaConfiguration } from '@daytona/api-client'
 import { useSuspenseQuery } from '@tanstack/react-query'
-import { InMemoryWebStorage, WebStorageStateStore } from 'oidc-client-ts'
+import { WebStorageStateStore } from 'oidc-client-ts'
 import { ReactNode, useMemo } from 'react'
 import { AuthProvider, AuthProviderProps } from 'react-oidc-context'
 import { ConfigContext } from '../contexts/ConfigContext'
@@ -32,7 +32,7 @@ export function ConfigProvider(props: Props) {
 
   const oidcConfig: AuthProviderProps = useMemo(() => {
     const isLocalhost = window.location.hostname === 'localhost'
-    const stateStore = isLocalhost ? window.sessionStorage : new InMemoryWebStorage()
+    const stateStore = isLocalhost ? window.sessionStorage : window.localStorage
 
     return {
       authority: config.oidc.issuer,
